src/modules/shared/components/NotificationCenter/NotificationCenter.tsx

```diff
@@ -59,40 +175,56 @@ const NotificationCenter: FC<NotificationCenterProps> = ({
 					)}
 					key={`notification-${notification.id}`}
 				>
-					<b
-						className={clsx(
-							'u-font-size-14',
-							styles['c-notification-center__notification-title']
-						)}
-					>
-						{notification.title}
-					</b>
-					{notification.description}
+					{renderNotificationLink(notification)}
 					<Button
-						onClick={() => onClickNotification(notification.id)}
+						onClick={() => onMarkOneAsRead(notification.id)}
 						className={clsx(styles['c-notification-center__notification-icon'])}
-						title="check"
+						title={t(
+							'modules/shared/components/notification-center/notification-center___markeer-als-gelezen'
+						)}
 						icon={<Icon name="check" />}
 						variants="text"
 					/>
 				</div>
 			);
-		});
+		} else {
+			return (
+				<div
+					className={clsx(
+						styles['c-notification-center__notification'],
+						styles['c-notification-center__notification--read']
+					)}
+					key={`notification-${notification.id}`}
+				>
+					{renderNotificationLink(notification)}
+				</div>
+			);
+		}
 	};
 
 	const renderFooter = () => (
 		<Button
 			disabled={!unreadNotifications.length}
-			onClick={onClickButton}
+			onClick={onMarkAllAsRead}
 			className={styles['c-notification-center__button']}
 			variants={['black', 'block']}
 			iconStart={<Icon name="check" />}
+			title={
+				unreadNotifications.length > 0
+					? t(
+							'modules/shared/components/notification-center/notification-center___markeer-alle-notificaties-als-gelezen'
+					  )
+					: t(
+							'modules/shared/components/notification-center/notification-center___alle-notificaties-zijn-reeds-gelezen'
+					  )
+			}
 			label={t(
 				'modules/shared/components/notification-center/notification-center___markeer-alles-als-gelezen'
 			)}
 		/>
 	);
 
+	const scrollableHeight = (windowSize?.height || 929) - 88 - 93; // Header and mark all as read button
```

Comment:
Hier kan je de `use-element-size` hook gebruiken. Die geeft je dan via een resizeObserver de dimensies van je element terug. Er wordt normaliter maar één resizeObserver gedefinieerd voor de hele app, dus dit lijkt me performanter dan een nieuwe `resize` listener op de window. 
Indien je dat liever niet gebruikt zou ik `use-window-size-context` gebruiken en de provider ook rond de notificaties zetten. In een toekomstige pr wordt die al in de `appLayout` rond de content gezet.

Reply:
Ik heb het eens geprobeerd met use-element-size maar dan lijkt hij niet te reageren op een window resize:
![image](https://user-images.githubusercontent.com/1710840/157448629-58a7b979-a329-4244-b3c5-fa2b92492027.png)

Met die window listener werkt het wel correct:
```
	const windowSize = useWindowSize();
```

Ik heb mijn changes om use-element-size in een aparte branch gecommit: feature/ARC-455-notifications-backend-integration-2
Voorlopig laat ik het "as-is" met use-window-size werken.